strings/strings.c

```diff
@@ -29,6 +29,26 @@ int string_length(char s[])
 */
 char *reverse_string(char rv[], char s[])
 {
+    // initialize length, begin and end variable to 0
+    int length, begin, end = 0;
+    // loop through until the value of string array is \0 char
+    while (s[length] != '\0')
```

Comment:
you can also just utilize the string_length() function.  Don't forget the brackets